inst/fill/fill.css

```diff
@@ -6,10 +6,16 @@
   min-height: 0;
 }
 .html-fill-container > .html-fill-item {
+  /* Fill items can grow and shrink freely within
+     available vertical space in fillable container */
   flex: 1 1 auto;
   overflow: auto;
   width: 100%;
 }
+.html-fill-container > :not(.html-fill-item) {
+  /* Prevent shrinking or growing of non-fill items */
+  flex: 0 0 auto;
+}
```

Comment:
I thought we were also going to have `min-height: 0`?

Reply:
I removed the `min-height: 0`. I couldn't construct an example where it mattered and after reading for a while I'm pretty certain it isn't relevant for our use case. This SO was helpful – https://stackoverflow.com/a/36247448/2022615 – especially the reiteration of this flex rule:

> A flex item cannot be smaller than the size of its content along the main axis.

So, as I understand it, setting `min-height: 0` allows the flex child be smaller than its content, but we already have `flex-shrink: 0`, so it wouldn't be allowed to shrink anyway unless you add `height: X`, since that's the another way to get around the above rule. But that's literally how we want people to adjust the height of the non-fill children anyway, so it all works out.